_posts/2016/04/2016-04-23-gsoc.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+authors: ["Raniere Silva"]
+title: "Welcome to Google Summer of Code Students"
+date: 2016-04-23
+time: "18:00:00"
+category: ["Community"]
+---
+Thanks to a lot of hard work from some members of our community and members of
+others open source scientific projects we are happy to welcome the students and
+mentors that will participate on Google Summer of Code this year under NumFOCUS
+umbrela.
+
+<table>
+<tbody>
+<tr>
+<th>Student</th>
+<th>Project</th>
+<th>Mentors</th>
+</tr>
+<tr>
+<td>Bhargav Srinivasa</td>
+<td>Dynamic Topic Models for <a href="https://github.com/piskvorky/gensim/">Gensim</a></td>
+<td>Lev Konstantinovskiy, Radim Rehurek and Devasena Inupakutika</td>
+</tr>
+<tr>
+<td>Chris Medrela</td>
+<td>Manage workflow for <a href="http://software-carpentry.org/">Software Carpentry</a> and <a href="http://datacarpentry.org/">Data Carpentry</a> instructor training</td>
+<td>Greg Wilson and Piotr Banaszkiewicz</td>
+</tr>
+<tr>
+<td>Akash Goel</td>
+<td>Upgrade to datapackage.json standard for <a href="https://github.com/weecology/retriever">EcoData Retriever</a></td>
+<td>Henry Senyondo and Ethan White</td>
+</tr>
+<tr>
+<td>Patrick Kofod Mogensen</td>
+<td>Improving the state of Optim.jl for <a href="http://juliaopt.org/">JuliaOpt</a></td>
+<td>Miles Lubin</td>
+</tr>
+<tr>
+<td>Prerit Garg</td>
+<td>Result-aggregation server for the installation-test scripts for <a href="http://software-carpentry.org/">Software Carpentry</a> and <a href="http://datacarpentry.org/">Data Carpentry</a></td>
+<td>Piotr Banaszkiewicz and Raniere Silva</td>
+</tr>
+<tr>
+<td>Ramchandran Muthukumar</td>
+<td>Presolve Routines for LP and SDP within Convex.jl for <a href="http://juliaopt.org/">JuliaOpt</a></td>
+<td>Madeleine Udell</td>
+</tr>
+<tr>
+<td>Hannah Aizenman</td>
+<td>Categorical Axis for <a href="http://matplotlib.org/">matplotlib</a></td>
+<td>Michael Droettboom and Thomas Caswell</td>
+</tr>
+</tbody>
+</table>
+
+This is the second year that NumFOCUS participate on Google Summer of Code and they have more than the double of slots from last year. Selecting the students was very difficult due the high quality of the proposals that are archived on [GitHub](https://github.com/numfocus/gsoc/tree/master/2016/proposals).
```

Comment:
"participate on" -> "has participated in"

"more than the double of" -> "more than doubled the number of"

"proposals that are" -> "proposals. The proposals are"